### PR TITLE
Adding splunk hec exporter tests

### PR DESCRIPTION
**Pull request by pszkamruk-splunk**
Description: 
Add a test suite to the opentelemetry-contrib repository that checks the behavior of the HEC exporter against a Splunk Enterprise instance.

Link to tracking Issue:
N/A

Testing:
added HEC exporter test pipeline with 19 tests

Documentation:
no changes needed

### CODE REVIEW

File: exporter/splunkhecexporter/test/requirements.txt
Comment (atoulme) [inline exporter/splunkhecexporter/test/requirements.txt:1]: Those tests probably will move to live with the e2e tests that @fatsheep9146 is working on.

**Review by atoulme — COMMENTED** [review]
Nice start! Please see my comments, mostly nits. Good chance those tests will be moved to e2e tests.

**Comment by dmitryax** [comment]
I don't think we should introduce python code in the contrib. The tests should be aligned with the testing suites that the contrib already has. Otherwise, it significantly increases the maintenance cost. 

We have integration tests per component, e2e tests for the k8s story. For this kind of test, there is https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/testbed. We should use that instead.

**Comment by pszkamruk-splunk** [comment]
Closing this PR - will not be merged
tests will be rewritten into golang and new PR will be created

**Pull request closed without merging**